docs/ros/ros/tutorials/driving.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Driving a robot
+sidebar_label: Teleop
```

Comment:
Should we keep this sidebar label a little closer to the "Driving a robot" title? I suggest we change one or the other. Slight preference to changing the sidebar to "Driving a Robot".